### PR TITLE
PalindromeChecker

### PR DESCRIPTION
**Pull request by SriRaghavi11**

### CODE REVIEW

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace PalindromeChecker {
+   class Program {
+      static bool IsPalindrome (string input) {
+         input = input.ToLower ().Replace (" ", "").Replace (".", "").Replace (",", "").Replace ("?", "").Replace ("'", "").Replace ("!", "");
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs:5]: Do we need to remove these special characters from the input string?

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+         int left = 0;
+         int right = input.Length - 1;
+
+         while (left < right) {
+            if (input[left] != input[right]) {
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs]: The enclosing brackets can be removed if there is only one statement inside them
for eg.
if (input[left] != input[right] 
   return false;

File: ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+         }
+
+         return true;
+      }
+
+      static void Main (string[] args) {
+      Console.WriteLine ("Enter a sentence or word:");
+      string input = Console.ReadLine ();
+
+         if (IsPalindrome (input)) {
```
Comment (asfar-nazar) [inline ConsoleApp1/Program.cs:19]: Use correct alignment. Here the 'if' block is 'tabbed' right. 
One thing you can do is type "CTRL + k + d" shortcut to apply the coding style automatically. Ping me if you have doubts regarding this.

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs:5]: In the text mentioned in example of assignment the sentence was "don't nod" so only I thought the need to remove the punctuations from sentence

File: ConsoleApp1/Program.cs (hunk 2 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs]: Will update the changes.

File: ConsoleApp1/Program.cs (hunk 3 above)
Comment (SriRaghavi11) [reply ConsoleApp1/Program.cs:19]: thanks for notifying the corrections .I will make the corrections and push again after making corrections

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (asfar-nazar) [reply ConsoleApp1/Program.cs:5]: Yes, the assignment states that the input can be a full sentence with special characters. You just have to check if the entire input string is a palindrome.

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**